build/yamls/antrea-coverage.yml

```diff
@@ -0,0 +1,1343 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Do we need to check-in these test-specific YAML files? I think we should just generate them dynamically with `generate-manifest.sh` before running the tests, as needed. Then we can avoid checking-in yet another set of large manifest files and having to keep them up-to-date.

Reply:
I heard from Zhecheng that when run jenkins e2e, we do not generate these yamls dynamically with generate-manifest.sh , in the jenkins job, we copy all the yaml files under build/yamls to the master node, so I generated the yaml files for coverage. I will discuss with Zhecheng if we could change in jenkins job.